salt/modules/win_lgpo.py

```diff
@@ -5655,23 +5662,25 @@ def _getAdmlPresentationRefId(adml_data, ref_id):
     if search_results:
         for result in search_results:
             the_localname = etree.QName(result.tag).localname
-            presentation_element = PRESENTATION_ANCESTOR_XPATH(result)
-            if presentation_element:
-                presentation_element = presentation_element[0]
-                if TEXT_ELEMENT_XPATH(presentation_element):
-                    for p_item in presentation_element.getchildren():
-                        if p_item == result:
-                            break
-                        else:
-                            if etree.QName(p_item.tag).localname == 'text':
-                                if prepended_text:
-                                    prepended_text = ' '.join((text for text in (prepended_text, getattr(p_item, 'text', '').rstrip()) if text))
-                                else:
-                                    prepended_text = getattr(p_item, 'text', '').rstrip() if getattr(p_item, 'text', '') else ''
-                            else:
-                                prepended_text = ''
-                    if prepended_text.endswith('.'):
-                        prepended_text = ''
+            # presentation_element = PRESENTATION_ANCESTOR_XPATH(result)
```

Comment:
We should remove code, not comment it out.

Reply:
I put that back in, but commented out in case we found the policy that code was written for.

Reply:
But, we can dig that out of an older release.